lte/gateway/c/core/oai/tasks/mme_app/mme_app_ip_imsi.cpp

```diff
@@ -77,11 +77,11 @@ int mme_app_get_imsi_from_ipv4(uint32_t ipv4_addr, imsi64_t** imsi_list) {
   } else {
     uint8_t idx = 0;
     num_imsis   = itr->second.size();
-    *imsi_list  = (imsi64_t*) calloc(num_imsis, sizeof(imsi64_t));
+    (*imsi_list)  = (imsi64_t*) calloc(num_imsis, sizeof(imsi64_t));
```

Comment:
Wow nice catch, another stylistic way I have seen is that you create a local imsi_list_ = nullptr and then reassign *imsi_list_ = imsi_list_ when we exit scope and only work with imsi_list_ so it syntactically looks better.